testing/next/APKBUILD

```diff
@@ -0,0 +1,24 @@
+# Maintainer: Will Sinatra <wpsinatra@gmail.com>
+# Contributor: Will Sinatra <wpsinatra@gmail.com>
+# Contributor: Benjamin Buccianti <bebuccianti@gmail.com>
+pkgname=next
+pkgver=1.3.2
+pkgrel=0
+pkgdesc="Atlas Engineer Next Browser"
+url="https://next.atlas.engineer"
+arch="x86_64"
```

Comment:
Why is it only available on x86_64?

Reply:
Because at this time SBCL is only built for x86_64. I haven't been able to get it to built on the arm platforms due to issues with CLISP. The long term plan is to fix that, then update any existent lisp package with an SBCL dependency to other platforms.

Reply:
Please add a comment briefly explaining that to the APKBUILD then

Reply:
All three should now be handled.